manifests/mailer.pp

```diff
@@ -2,13 +2,30 @@
 #
 # Sensu handler to send emails.
 #
-class sensu_handlers::mailer inherits sensu_handlers {
+class sensu_handlers::mailer (
+  $dependencies = {}
+) inherits sensu_handlers {
 
-  ensure_packages(['nagios-plugins-basic'])
+  if $dependencies {
+
+    $defaults = {
+      'nagios-plugins-basic' => {
```

Comment:
Exclude this for now in this first pass. On a second pass this can be replaced with a sensu gem.